src/Query/TcpTransportExecutor.php

```diff
@@ -252,8 +250,8 @@ public function handleWritable()
             // Match errstr from PHP's warning message.
             // fwrite(): Send of 327712 bytes failed with errno=32 Broken pipe
             \preg_match('/errno=(\d+) (.+)/', $error, $m);
-            $errno = isset($m[1]) ? (int) $m[1] : 0;
-            $errstr = isset($m[2]) ? $m[2] : $error;
+            $errno = (int) $m[1] ?? 0;
```

Comment:
Casting operator precedence requires additional parenthesis:

```suggestion
            $errno = (int) ($m[1] ?? 0);
```

Reply:
Thanks for the hint, fixed it :+1: